src/en/clouds-LXD.md

```diff
@@ -1,30 +1,16 @@
 Title: Juju LXD local provider
 
-# Configuring for LXD
+# Using LXD as a cloud provider 
 
-Here we provide an overview for the creation of a controller for LXD
-(see [Controllers][bootstrap]). If your objective is instead
-to create a LXD model please see [Adding a model][models-add].
-
-Unlike other providers, with LXD, Juju does not need to be supplied with any
-information regarding configuration or credentials. This makes it extremely
-easy to start using LXD with Juju.
-
-!!! Note: Do not confuse command `lxc` with the binary shipped with traditional
-LXC. All the latter's binaries are of the form `lxc-<subcommand>`. The `lxc`
-binary actually comes from the `lxd-client` package.
+LXD provides a fast, powerful, self-contained and largely configuration-free 
+way to experiment with Juju. Using lightweight LXC containers as instances,
```

Comment:
Are they still called "LXC containers" or should this also be updated to "LXD containers" ?